lib/ood_core/job/info.rb

```diff
@@ -120,7 +131,8 @@ def to_h
           cpu_time:        cpu_time,
           submission_time: submission_time,
           dispatch_time:   dispatch_time,
-          native:          native
+          native:          native,
+          child_task_statuses: child_task_statuses
```

Comment:
Maybe we should rename this to be: `child_tasks` or just `tasks`. And right now a task only contains a status and an id, but it is a bucket for other possible relevant attributes in the future and then the method naming makes more sense to me: `OodCore::Job::Task#status`  - returns the status of an `OodCore::Job::Task` sounds better than `OodCore::Job::TaskStatus#status` returns the status of an `OodCore::Job::TaskStatus`

Though its true that is not what we previously discussed. Do you see any drawbacks to having a "Task" object over a "TaskStatus" object?

Reply:
I'm good with the alternate naming.